FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -62,6 +62,21 @@ private void executeTest(string connString, string commandText, string? addition
             }
         }
 
+        [TestCase("select 1", 1)]
+        [TestCase("select 'hello'", "hello")]
+        [TestCase("select 2, 'two'", 2)]
+        [TestCase("select 'three', 3", "three")]
+        [TestCase("select table_schema, table_name from information_schema.tables where table_name = 'tables'", "information_schema")]
+        [TestCase("select table_name, table_schema from information_schema.tables where table_schema = 'information_schema' order by table_schema", "accounts")]
```

Comment:
I wouldn't rely on `information_schema`, since we don't control what's in it. Something like `SELECT 'one', 2 UNION ALL SELECT 'two', 3` will give a nice 2x2 result and will almost 100% stay the same.